simple_history/models.py

```diff
@@ -349,12 +356,14 @@ def get_history_user(self, instance):
         try:
             return instance._history_user
         except AttributeError:
+            request = None
             try:
                 if self.thread.request.user.is_authenticated:
-                    return self.thread.request.user
-                return None
+                    request = self.thread.request
             except AttributeError:
-                return None
+                pass
+
+        return self.get_user(history=self, instance=instance, request=request)
```

Comment:
Why include the HistoricalRecords instance in `self.get_user`? What might need to use that?

Reply:
Once I hear back about this `history` here, I'm happy to merge.

Reply:
I thought it may be useful to give the historical object to the method since it is a generic API that the user is providing.  It could be removed.

Reply:
Yea I don't see a reason for it to be included. Let's get rid of it for now and if someone ends up needing it, we can reevaluate. Just remove that and then I'm happy to merge.